templates/download/shared/_verify-checksums.html

```diff
@@ -3,13 +3,17 @@
   <span class="p-contextual-menu--left">
     <a href="#" class="p-contextual-menu__toggle" data-trigger="click" aria-controls="#menu-4" aria-expanded="false" aria-haspopup="true">verify your download</a>
     <span class="p-contextual-menu__dropdown" id="menu-4" aria-hidden="true" aria-label="Verification instructions:" style="width: 600px; max-width: 70vw; padding: 1rem;">
+      {% if releases.checksums[system][version] %}
       <span class="p-contextual-menu__group">
         <small>Run this command in your terminal in the directory the iso was downloaded to verify the SHA256 checksum:</small><br />
         <code style="display: block; margin-top: 1rem;">echo "{{ releases.checksums[system][version] }}" | shasum -a 256 --check</code><br />
         <small>You should get the following output:</small><br />
         <code style="display: block; margin-top: 1rem;">ubuntu-{{ version }}-{% if system != architecture %}{{ system }}-{% else %}preinstalled-server-{% endif %}{{ architecture }}.iso: OK</code><br />
         <small>Or follow this tutorial to learn <a class="p-link--external" href="/tutorials/tutorial-how-to-verify-ubuntu">how to verify downloads</a></small>
       </span>
+      {% else %}
+        <small>Please check the <a class="p-link--external" href="http://releases.ubuntu.com/{{ version }}/SHA256SUMS">SHA256SUMS page</a> for the checksum numbers.</small>
```

Comment:
Shouldn't this still include the instructions? Or a link to a tutorial?

Reply:
There is a tutorial for all but the Pi, which appears for them, not sure what to do for Pi today.